dev-util/conan/conan-1.36.0.ebuild

```diff
@@ -37,22 +37,28 @@ RDEPEND="
 	>=dev-python/requests-2.8.1[${PYTHON_USEDEP}]
 	>=dev-python/six-1.12.0[${PYTHON_USEDEP}]
 	>=dev-python/tqdm-4.28.1[${PYTHON_USEDEP}]
-	>=dev-python/urllib3-1.25.7[${PYTHON_USEDEP}]
+	>=dev-python/urllib3-1.25.8[${PYTHON_USEDEP}]
 "
 
-distutils_enable_tests nose
-DEPEND+="
-	test? (
-		dev-lang/go
-		dev-vcs/git
-		dev-vcs/subversion
-		>=dev-python/mock-1.3.0[${PYTHON_USEDEP}]
-		>=dev-python/nose-1.3.7[${PYTHON_USEDEP}]
-		>=dev-python/parameterized-0.6.3[${PYTHON_USEDEP}]
-		>=dev-python/webtest-2.0.18[${PYTHON_USEDEP}]
-		dev-util/cmake
-	)
-"
+# Try to fix it if you're brave enough
+# Conan requires noumerous external toolchain dependencies with restricted
+# versions and cannot be managable outside of a pure CI environment.
+RESTRICT="test"
+
+#distutils_enable_tests pytest
+#DEPEND+="
+#	test? (
+#		dev-lang/go
+#		dev-vcs/git
+#		dev-vcs/subversion
+#		>=dev-python/mock-1.3.0[${PYTHON_USEDEP}]
+#		>=dev-python/parameterized-0.6.3[${PYTHON_USEDEP}]
+#		>=dev-python/pytest-6.1.1[${PYTHON_USEDEP}]
+#		dev-python/pytest-xdist[${PYTHON_USEDEP}]
+#		>=dev-python/webtest-2.0.18[${PYTHON_USEDEP}]
+#		dev-util/cmake
+#	)
+#"
```

Comment:
Remove instead of commenting out.  The brave soul wanting to fix the tests can find this in the Git history.